lib/main.js

```diff
@@ -66,11 +66,16 @@ const main = module.exports = async(linkOrId, options, rt = 3) => {
   };
 
   // Parse videos
-  const rawVideoList = parsed.json.contents
+  const playlistVideoListRenderer = parsed.json.contents
     .twoColumnBrowseResultsRenderer.tabs[0].tabRenderer.content
     .sectionListRenderer.contents[0]
     .itemSectionRenderer.contents[0]
-    .playlistVideoListRenderer.contents;
+    .playlistVideoListRenderer;
```

Comment:
We could use optional chaining here but that requires node v14.5.0 or above, so I used the ternary operator to be compatible with node v8. I do recommend you bump node to v14.5.0 or above because v8 is EOL and v12 is going to EOL next year

Reply:
With optional chaining the updated code should look like this
```js
const rawVideoList = parsed.json.contents
    .twoColumnBrowseResultsRenderer.tabs[0].tabRenderer.content
    .sectionListRenderer.contents[0]
    .itemSectionRenderer.contents[0]
    .playlistVideoListRenderer?.contents;

if (rawVideoList === undefined) throw new Error('Playlist is empty');
```

Reply:
With the current changes, it would be like this with optional chaining

```js
  const playlistVideoListRenderer = parsed.json.contents
    .twoColumnBrowseResultsRenderer.tabs[0].tabRenderer.content
    .sectionListRenderer.contents[0]
    .itemSectionRenderer.contents[0]
    .playlistVideoListRenderer?.contents ?? [];
  ```